scan-engine/src/main/java/de/tillhub/scanengine/data/ScannerEvent.kt

```diff
@@ -3,6 +3,7 @@ sealed class ScannerEvent {
     sealed class External : ScannerEvent() {
         data object NotConnected : External()
         data object Connected : External()
+        data object InProgress : External()
```

Comment:
InProgress is misleading especially when you compare with InProgress that we have in Camera. Rename it to `Connecting`

Reply:
Maybe this should be data class which contains device id or something similar.